clients/web/src/views/body/UsersView.js

```diff
@@ -60,14 +59,27 @@ var UsersView = View.extend({
             parentView: this
         }).on('g:resultClicked', this._gotoUser, this);
 
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
+            const userCountPromise = UserCollection.getTotalCount()
+                .done((count) => {
+                    this.usersCount = count;
+                });
+            promiseArray.push(userCountPromise);
+        }
         this.register = settings.dialog === 'register' && getCurrentUser() &&
                         getCurrentUser().get('admin');
+
+        $.when(...promiseArray)
+            .done(() => {
+                this.render();
+            });
     },
 
     render: function () {
         this.$el.html(UserListTemplate({
             users: this.collection.toArray(),
-            getCurrentUser: getCurrentUser,
+            currentUser: getCurrentUser,
```

Comment:
The Ci test failed because of that : 

![screenshot from 2017-08-10 18-08-05](https://user-images.githubusercontent.com/29961430/29194292-e615b262-7df6-11e7-80c1-59402def786a.png)

Delete this line break the event of the sort widget button, It's working if I let them, but I think it's not clean at all so I wait you advice on this point.